ch_backup/logic/named_collections.py

```diff
@@ -0,0 +1,74 @@
+"""
+Clickhouse backup logic for named collections
+"""
+from typing import List
+
+from ch_backup import logging
+from ch_backup.backup_context import BackupContext
+from ch_backup.logic.backup_manager import BackupManager
+
+
+class NamedCollectionsBackup(BackupManager):
+    """
+    Named collections backup class
+    """
+
+    def backup(self, context: BackupContext) -> None:
+        """
+        Backup named collections.
+        """
+        if not context.ch_ctl.ch_version_ge("23.2"):
```

Comment:
If ch_version is less than `23.2` then we quit silently? Maybe to put some log msg here?

Reply:
Done. And actually method works from 22.12, in 23.2 only was added grants for user.